src/Core/src/Platform/MauiContext.Windows.cs

```diff
@@ -2,11 +2,23 @@
 
 namespace Microsoft.Maui
 {
-	public partial class MauiContext
+	public partial class MauiApplicationContext
```

Comment:
internal for now? Since we're making the interfaces internal?

Reply:
Doing a total re-work of these things. Thanks.